src/util/server.c

```diff
@@ -74,6 +74,12 @@ static void become_daemon(void)
     int ret, error;
 
     pid = fork();
+    if (pid == -1) {
+        DEBUG(SSSDBG_FATAL_FAILURE, "fork() failed: %d [%s]\n",
```

Comment:
Please, use `ret = errno` and then refer to `ret` only since in theory something in `DEBUG` macro can overwrite `errno` value.

Reply:
Thank you, fixed.